_data/sandeepcoder097/sandeepcoder097.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Sandeep Sharma
+institution: Galgotias University
+profile_pic: sandeepcoder097.png
+quote: "An investment in knowledge pays the best interest" 
```

Comment:
Remove double quotes from begin and end.

Reply:
done